omniscidb/QueryEngine/Execute.cpp

```diff
@@ -14,8 +14,10 @@
  * limitations under the License.
  */
 
-#include "QueryEngine/Execute.h"
+#include "Shared/funcannotations.h"
+#define QUERYENGINE_EXPORT RUNTIME_EXPORT
```

Comment:
I'd add an empty line after such defines to not hide them within a bunch of includes. Otherwise looks good.